src/modules/RA/DataLayer/components/tabs/PopupTab/index.js

```diff
@@ -47,27 +57,49 @@ const PopupTab = () => {
   const form = useForm();
   const classes = useStyle();
 
-
   const updateTemplate = useCallback(debounce(() => {
     if (popupfields.length) {
-      const lines = popupfields.map(({
+      const [titlefield, ...contentfields] = popupfields;
+      const titleLine = (
+        `# {% if ${titlefield.field.name} %}`
+        + `{{${titlefield.field.name}}}`
+        + `{% else %}${titlefield.default}{% endif %}`
+      );
+
+      const lines = contentfields.map(({
         suffix,
         prefix,
-        field: { name },
+        field: { name, label },
         default: defaultTitle,
-      }) => (
-        `${prefix}: {% if ${name}.length > 0} %}{{${name}}} ${suffix}{% else %}${defaultTitle}{% endif %}`
-      ));
+        sourceFieldId,
+      }) => {
+        const { settings: { round } } = fields.find(f => (f.sourceFieldId === sourceFieldId));
+        if (round !== undefined) {
+          return (
+            `- ${label} : ${prefix} `
```

Comment:
Prefix part should be inside if, like the suffix.

Reply:
Done.